xml/System.Net/WebRequest.xml

```diff
@@ -710,7 +710,7 @@
  For example, when a URI beginning with `http://` is passed in `requestUri`, an <xref:System.Net.HttpWebRequest> is returned by <xref:System.Net.WebRequest.CreateDefault%2A>. If a URI beginning with `file://` is passed instead, the <xref:System.Net.WebRequest.CreateDefault%2A> method will return a <xref:System.Net.FileWebRequest>.  
   
 > [!NOTE]
->  This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in the .NET Framework](~/docs/framework/network-programming/network-tracing.md).  
+>  This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in .NET](~/docs/framework/network-programming/network-tracing.md).  
```

Comment:
This looks good. Side note: we might have to move this content from under the .NET Framework Guide to the .NET Guide. We distinguish cross-platform content that way.

Reply:
The PR is showing "changes requested", but I don't see them. I see a "looks good". :)